docs/ja/user-guide/api.md

```diff
@@ -48,29 +50,39 @@ Content-Type: application/json
 Authorization: Bearer <YOUR_TOKEN_HERE>
 ```
 
-Example request:
-![Postman response](../../../user-guide/assets/postman.png)
+リクエストの例:
+![Postman response](../../user-guide/assets/postman.png)
 
 詳しい情報と例についてはAPIドキュメントをご覧ください。
 
 ## 認可と認証
 
-認証のために、[JSON Web Tokens (JWT)](http://jwt.io)を使用しています。JWTは
-`Authorization`ヘッダを必要とします。JWTを生成するために`/v4/auth/login` にアクセスし、`/v4/auth/token`へとリダイレクトされます。
+認証のために、[JSON Web Tokens (JWT)](http://jwt.io)を使用しています。JWTは`Authorization`ヘッダを必要とします。JWTを生成するために`/v4/auth/login` にアクセスし、`/v4/auth/token`へとリダイレクトされます。
 
 一方、認可はOAuthによるものです。`/v4/auth/login`にアクセスしたときに行われます。ScrewdriverはSCMトークンで以下を識別します。
 
 - レポジトリへのread, write, adminアクセスを識別します。
-    - read権限でpipelineを見ることができます。
-    - write権限でjobの開始と停止ができます。
-    - admin権限でpipelineの作成、編集、削除ができます。
 - リポジトリの`screwdriver.yaml`の読み込み
-- ビルドの成功・失敗情報でpull-requestを更新
+- リポジトリに対するオープン中のpull-requestのリストを取得
 - ビルドの成功・失敗情報でpull-requestを更新
 - Screwdriverが変更の通知を受け取れるよう、リポジトリに対しwebhookを追加・削除
 
 より詳しい情報については[GitHub OAuth](https://developer.github.com/v3/oauth/)のドキュメントをご覧ください。
 
+## バッジ
+
+`<your_UI_URL>/pipelines/<your_pipelineId>/badge`のURLを利用して、特定のパイプラインに対する現在のビルドステータスを表す画像を取得できます。
+
+[![Build Status](https://cd.screwdriver.cd/pipelines/1/badge)](https://cd.screwdriver.cd/pipelines/1)
+
+例えば、このMarkdown形式で書かれたコードを利用することで上のようなバッジを表示できます。`status-image` URLはバッジの画像で、`status-url` はパイプラインへのリンクです。
+
+```markdown
+[![Build Status][status-image]][status-url]
+[status-image]: https://cd.screwdriver.cd/pipelines/1/badge
```

Comment:
Need blank line between `[![Build Status][status-image]][status-url]` and `[status-image]: https://cd.screwdriver.cd/pipelines/1/badge`